packages/@aws-cdk/core/test/test.duration.ts

```diff
@@ -107,7 +123,24 @@ export = nodeunit.testCase({
     test.equal(Duration.parse('PT1D1H1M1S').toSeconds(), 1 + 60 * (1 + 60 * (1 + 24)));
 
     test.done();
-  }
+  },
+
+  'to human string'(test: nodeunit.Test) {
+    test.equal(Duration.minutes(0).toHumanString(), '0 minutes');
+    test.equal(Duration.minutes(Lazy.numberValue({ produce: () => 5 })).toHumanString(), '<token> minutes');
+
+    test.equal(Duration.minutes(10).toHumanString(), '10 minutes');
+    test.equal(Duration.minutes(1).toHumanString(), '1 minute');
+
+    test.equal(Duration.minutes(62).toHumanString(), '1 hour 2 minutes');
+
+    test.equal(Duration.seconds(3666).toHumanString(), '1 hour 1 minute');
+
+    test.equal(Duration.millis(3000).toHumanString(), '3 seconds');
+    test.equal(Duration.millis(3666).toHumanString(), '3 seconds 666 millis');
```

Comment:
Does it do the right thing for `Duration.millis(3.6).toHumanString()`?

Reply:
Probably not. You crazy person.